src/components/apps/launch/ResourceRequirements.js

```diff
@@ -127,8 +127,8 @@ const StepResourceRequirementsForm = ({
                 <Grid item xs={12}>
                     <FastField
                         id={buildID(baseId, ids.RESOURCE_REQUESTS.TOOL_CPU)}
-                        name={`requirements.${index}.min_cpu_cores`}
-                        label={t("minCPUCores")}
+                        name={`requirements.${index}.max_cpu_cores`}
+                        label={t("cpuCores")}
```

Comment:
I would also remove the `selectMins` grid item just above this, since these are no longer all minimum requirements.